src/libraries/System.Drawing.Common/src/System/Drawing/BufferedGraphics.Unix.cs

```diff
@@ -37,18 +37,18 @@ namespace System.Drawing
     public sealed partial class BufferedGraphics
     {
         private Rectangle size;
-        private Bitmap membmp = null;
-        private Graphics source = null;
+        private Bitmap membmp;
+        private Graphics? source;
 
-        internal BufferedGraphics(Graphics targetGraphics, IntPtr targetDc, Rectangle targetRectangle)
+        internal BufferedGraphics(Graphics? targetGraphics, IntPtr targetDc, Rectangle targetRectangle)
         {
             _targetGraphics = targetGraphics;
             _targetDC = targetDc;
             size = targetRectangle;
             membmp = new Bitmap(size.Width, size.Height);
         }
 
-        public Graphics Graphics
+        public Graphics? Graphics
```

Comment:
Should this be nullable? It seems like the only way we could return a null Graphics object is after dispose, which to me is an invalid usage of this property. Probably, we should make this non-nullable and bang away the warning in the return.

Reply:
You're right. I'll change this.